packages/haul-core/src/server/UserInterface.ts

```diff
@@ -99,7 +99,7 @@ export default class UserInterface {
 
     platforms.forEach((platform, index) => {
       this.compilations[platform] = {
-        width: Math.min(110, this.terminal.width) - (platformNameLength + 20),
+        width: Math.min(110, this.terminal.width) - (platformNameLength + 22),
```

Comment:
this should go in a separate commit, but I guess you're aware :P

Reply:
Well ideally this should go into separate PR

Reply:
```suggestion
        width: Math.min(110, this.terminal.width) - (platformNameLength + 20),
```